com.microsoft.mrtk.graphicstools.unity/Editor/ShaderGUIs/BaseShaderGUI.cs

```diff
@@ -407,11 +407,46 @@ protected static bool PropertyEnabled(MaterialProperty property)
         /// <param name="material">material to check</param>
         /// <param name="propertyName">name of property against material</param>
         /// <returns>if has property, then value of that property for current material, null otherwise</returns>
-        protected static float? GetFloatProperty(Material material, string propertyName)
+        protected static float? GetFloatProperty(Material material, params string[] propertyNames)

```

Comment:
Feels like this family of static methods would be better homed in the ShaderUtils class